templates/node/lib/inputFile.js.twig

```diff
@@ -19,10 +19,14 @@ class InputFile {
     return new InputFile(stream, filename, size);
   };
 
-  static fromBlob = (blob, filename) => {
-    const buffer = blob.arrayBuffer();
-    const stream = Readable.from(buffer.toString());
-    const size = Buffer.byteLength(buffer);
+  static fromBlob = async (blob, filename) => {
+    const buffer = await blob.arrayBuffer();
+    const stream = new Readable({
+      read() {
+        this.push(buffer);
+        this.push(null);
+      }
+    });
     return new InputFile(stream, filename);
```

Comment:
If a stream is needed, why not use [`blob.stream()`](https://nodejs.org/docs/latest-v18.x/api/buffer.html#blobstream)?

```suggestion
  static fromBlob = async (blob, filename) => {
    const stream = blob.stream();
    const size = blob.size;
    return new InputFile(stream, filename, size);
```

Reply:
Yeah it could be used I guess, but it was only recently added
![image](https://user-images.githubusercontent.com/52203828/202575399-4fc08f5f-143f-4d3d-8a5c-5a2d5890070e.png)

This is how I usually convert blobs to streams, so that's what I've put here, but your method is probably better IMO.

Reply:
Hmm looks like Node 14 is EOL 2023-04-30 so we might be safe just support Node 16... @lohanidamodar, what are your thoughts?

Reply:
We discussed internally and decided to make sure we support node 14 at least until EOL.